Aquarius/Src/Aquarius/Events/Mouse.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include <GLFW/glfw3.h>
+#include "Aquarius/Core/Log.h"
+
+namespace Aquarius {
+    class MouseEvent {
+    public:
+        static void cursorPositionCallback(GLFWwindow* window, double xPosition, double yPosition)
+        {
+            // TODO: Discuss if we want to only keep track of cursor position when the cursor is on
+            //  the Application window, if so can combine this callback method and the one below.
+            AQ_INFO("x position: %v", xPosition);
+            AQ_INFO("y position: %v", yPosition);
+        }
+
+        static void cursorInWindowCallback(GLFWwindow* window, int inWindow)
+        {
+            if (inWindow)
```

Comment:
Seeing an integer interpreted as a boolean is slightly misleading, but this is a consequence of glfw being written in C (C doesn't have a bool type), so this is fine here